esf_main.tex

```diff
@@ -298,31 +298,39 @@ \subsubsection{Position in car}
 \subsection{Tractive system active light}
 \subsubsection{Description/circuitry}
 %Describe the tractive system active light and additional circuitry.
-TODO
+The TSAL illuminates when the tractive system is active, which is defined as the tractive system voltage being over 60V.
 
 	\begin{table}[H]
 	    \centering
 	    \begin{tabular}{|l|l|}
 	    \hline
 	    Supply voltage: & 12V \\ \hline
-	    Max. operational current: &  0.04A\\ \hline
-	    Lamp type & LEDs \\ \hline
-	    Power consumption: & 0.48 W\\ \hline
-	    Brightness & Unknown\\ \hline
-	    Frequency: & Manual with 555 timer, 2.4 Hz \\ \hline
-	    Size (length x height x width): & 103x27x51 mm \\ \hline
+	    Max. operational current: &  2A\\ \hline
+	    Lamp type: & LEDs \\ \hline
+	    Power consumption: & 24W (max)\\ \hline
+	    Brightness: & Unknown\\ \hline
+	    Frequency: & 2 Hz \\ \hline
+	    Size (length x height x width): & 41x41x32 mm \\ \hline
 	    \end{tabular}
 	    \caption{Parameters of the TSAL}
 	    \label{TSALparameters}
 	\end{table}
 
 \subsubsection{Wiring/cables/connectors}
 %Describe wiring, show schematics, describe connectors and cables used and show useful data regarding the wiring.  Include gauge, voltage and temperature rating of wiring used and any fuses or other overcurrent protection used.
-TODO
+The circuitry designed has a TS-controlled optocoupler. When the TS is over 60V, zener diodes with a breakdown voltage of 58V and 3.6V respectably power the photodiode side which switches the mosfet side allowing current to flow from the GLV to power the TSAL. The frequency of the TSAL will be controlled by built in settings to flash at 2Hz.
+
+\begin{figure}[h]
+\centering
+\includegraphics[scale=1]{TSAL}
+\caption{Schematic for the TSAL}
+\end{figure}
+
+All connections made by wires will be 20 AWG rated for 600V, 125°C and 7A, while all PCB traces will be a minimum of 12 mil, but nominally 20 mil in width. The TS voltage will be fused to 1A on the power line.
 
 \subsubsection{Position in car}
 %Provide CAD-renderings showing the relevant parts. Mark the parts in the rendering, if necessary.
-TODO
+The TSAL will be mounted to the underside of the highest point of the main roll hoop, per EV 4.12.3 & 4 using a robust 3D printed (Kevlar reinforced Nylon) bracket integrating the light and necessary wiring. This enclosure has not been designed yet. The PCB will be located in the master switch panel enclosure.
```

Comment:
do i need to request one from someone in mechanical? if so, who? if not, where can i find the rendering?